Source/Fuse.Vibration/Vibrate.uno

```diff
@@ -17,22 +17,61 @@ namespace Fuse.Vibration
 		  	}
 
 		## Example
-			
+
 			<StackPanel Margin="20">
 				<Button Margin="10" Text="Vibrate">
 					<Clicked>
 						<Vibrate Duration="5" />
 					</Clicked>
 				</Button>
 			</StackPanel>
+
+		On iOS you can do vibration by leveraging Taptic Engine. There are 9 types of vibration :
+		* Soft
+		* Rigid
+		* light
```

Comment:
`Light` should probably start with an upper-case letter, same as the other bullet points.